include/toplevel.mk

```diff
@@ -25,7 +25,7 @@ export SOURCE_DATE_EPOCH
 export GIT_CONFIG_PARAMETERS='core.autocrlf=false'
 export GIT_ASKPASS:=/bin/true
 export MAKE_JOBSERVER=$(filter --jobserver%,$(MAKEFLAGS))
-export GNU_HOST_NAME:=$(shell $(TOPDIR)/scripts/config.guess)
+export GNU_HOST_NAME:=$(shell $(TOPDIR)/scripts/get-gnu-host-name.sh)
```

Comment:
I think we just better try to fix config.guess. It looks like this commit it missing:
https://git.savannah.gnu.org/gitweb/?p=config.git;a=commitdiff;h=bd9626458c30d7faec17d7dfbd85a80617b10007

I would prefer if we just update config.guess and config.sub to the most recent version